src/main/java/com/minecolonies/structures/helpers/Structure.java

```diff
@@ -105,7 +105,14 @@ public Structure(@Nullable final World world, final String structureName, final
                 {
                     decorationFolder = new File(FMLCommonHandler.instance().getMinecraftServerInstance().getDataDirectory(), "minecolonies/");
                 }
-                inputstream = new FileInputStream(decorationFolder.getPath() + "/" + structureName + ".nbt");
+                if(decorationFolder.exists())
+                {
+                    inputstream = new FileInputStream(decorationFolder.getPath() + "/" + structureName + ".nbt");
+                }
+                else
+                {
+                    throw new FileNotFoundException();
```

Comment:
Will do on FileNotFound:
-  Log.getLogger().warn(String.format("Failed to load template %s", structureName), e);

Usually this only happens when the player or server owner removes a schematic a worker is trying to build.